### PR TITLE
LPS-135170

### PR DESCRIPTION
**Pull request by brianikim**
See slack thread: https://liferay.slack.com/archives/CL9RTSZ52/p1611845640071900

https://issues.liferay.com/browse/LPS-135170

Solution is a direct implementation of the one suggested by @ealonso in that thread.

Please update this PR as soon as possible as it is customer critical. Furthermore, please let me know if this can be provided in a hot-fix asap.

### CODE REVIEW

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
Rerunning CI:test since it's a new PR, but here's a successful test for this commit: https://github.com/dacousalr/liferay-portal/pull/341#issuecomment-957990244

File: portal-impl/src/com/liferay/portal/model/impl/LayoutTypePortletImpl.java [hunk 1]
```diff
@@ -473,6 +479,27 @@ public PortalPreferences getPortalPreferences() {
 				portletIds, StringUtil.split(getColumnValue(column)));
 		}
 
+		Layout layout = getLayout();
+
+		if (Objects.equals(
+				layout.getType(), LayoutConstants.TYPE_ASSET_DISPLAY) ||
```
Comment (jkappler) [inline portal-impl/src/com/liferay/portal/model/impl/LayoutTypePortletImpl.java:485]: Hi @brianikim, just mentioning, we have: layout.isTypeContent() and layout.isTypeAssetDisplay() that we can use here. That will cover also future cases.

Comment (brianikim) [reply portal-impl/src/com/liferay/portal/model/impl/LayoutTypePortletImpl.java:485]: hey @jkappler , I will update the PR as soon as CI:test finishes. Also, what do you think about the PR as a hot-fix?

**Comment by jkappler** [comment]
Hi @brianikim. looks good based on the slack thread that you mentioned. Still, we'll check it out tomorrow and most likely send it to core infra for a final review

Thanks again for the PR!

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
Just started reviewing :)

**Comment by jkappler** [comment]
@brianikim, please see https://github.com/shuyangzhou/liferay-portal/pull/10393

**Comment by jkappler** [comment]
Hi @brianikim, there are performance regressions (which I was afraid of) introduced in the PR. Please see https://github.com/shuyangzhou/liferay-portal/pull/10393#issuecomment-964071497

Could you please take a look? This reminded me of the PTR where we talked about it, and leave it as a limitation if there were performance issues.

Thanks  

@dacousalr

**Pull request closed without merging**